manifests/init.pp

```diff
@@ -137,7 +137,7 @@
   Enum['stopped', 'running']            $service_ensure              = 'running',
   Hash                                  $services                    = {},
   String[1]                             $user                        = $consul::params::user,
-  String[1]                             $version                     = '1.2.3',
+  String[1]                             $version                     = '1.16.1',
```

Comment:
I generally wonder if it is not better to _not_ set any default version, and mandate the user to explicitly put what they want.  This mean that we have to set an arbitrary version for testing, but we do not have to live with having to choose to default to an old-legacy-deprecated version after a few years or break backwards compatibility when a new version is available.

Reply:
I considered that and I don't know if I like it or not. We need to do a major release anyway so I thought it's a good time to bump the version, also consul supports upgrades between versions.